gitlab/__init__.py

```diff
@@ -521,3 +521,21 @@ def Team(self, id=None, **kwargs):
         to write it on the server.
         """
         return Team._get_list_or_object(self, id, **kwargs)
+
+    def Status(self, id=None, **kwargs):
```

Comment:
@gpocentek Do you recommend I simply delete this function?  I only created it because it seemed consistent with the other functions -- I was not sure if it was necessary.